app/assets/javascripts/template_invocation.js

```diff
@@ -0,0 +1,32 @@
+$(function() { job_invocation_form_binds() });
+
+function refresh_execution_form() {
+  var form = $('form#job_invocation_form');
+  var data = form.serializeArray();
+
+  request = $.ajax({
+    data: data,
+    type: 'POST',
+    url: form.attr('data-refresh-url'),
+    dataType: 'script'
+  });
+
+  request.done(function () {
+    password_caps_lock_hint();
+    form.find('a[rel="popover-modal"]').popover({html: true});
+    form.find('select').select2({allowClear: true});
+    job_invocation_form_binds();
+  });
+}
+
+function job_invocation_form_binds() {
+  $('input.job_template_selector').on('click', function () {
+    parent_fieldset = $(this).closest('fieldset');
```

Comment:
same as above, no such convention in Foreman, we have mixed style, could we keep it similar to Ruby in this plugin?